packages/hub/src/vendor/xet-chunk/chunker_wasm.ts

```diff
@@ -0,0 +1,62 @@
+// export * from "./chunker_wasm_bg.js";
+import * as __glue_imports from "./chunker_wasm_bg.js";
+// @ts-expect-error no types
+import { __wbg_set_wasm } from "./chunker_wasm_bg.js";
+// @ts-expect-error no types
+import { wasmBinary } from "./chunker_wasm_bg.wasm.base64.ts";
+
+let initPromise: Promise<void> | null = null;
+
+async function init(): Promise<void> {
+	if (initPromise) {
+		return initPromise;
+	}
+	let resolve: (value: void) => void;
+	let reject: (reason?: unknown) => void;
+	initPromise = new Promise((_resolve, _reject) => {
+		resolve = _resolve;
+		reject = _reject;
+	});
+
+	await Promise.resolve();
```

Comment:
A bit suspicious as a method to create a singleton you can't recreate if the wasm import is failing